sfepy/discrete/iga/extmods/igac.pyx

```diff
@@ -351,7 +351,7 @@ def eval_mapping_data_in_qp(np.ndarray[float64, mode='c', ndim=2] qps not None,
     dx_dxi = np.empty((1, 1, dim, dim), dtype=np.float64)
     dxi_dx = np.empty((1, 1, dim, dim), dtype=np.float64)
 
-    for ii in range(0, dim):
+    for ii in range(0, <uint32>dim):
```

Comment:
Did this generate warnings too?

Reply:
Yes... not from cython, but following generated c-code compilation.

Reply:
OK, thanks!